src/operations/find_and_modify.ts

```diff
@@ -140,6 +149,10 @@ class FindAndModifyOperation extends CommandOperation<Document> {
       this.cmdBase.let = options.let;
     }
 
+    if (options.comment) {
```

Comment:
I think our if stmt filters for comment should be more specific given the wide array of types that can be passed in, I don't see why number `0` shouldn't be allowed. But maybe even null should be permitted as well, this might be one of the few cases where checking explicitly for undefined is warrented. Thoughts?

Reply:
Addressed! d